grpc/client/src/main/java/io/helidon/grpc/client/util/SingleValueResponseStreamObserverAdapter.java

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Refactored it. Moved it as an inner class of GrpcServiceClient and renamed it as SingleResponseStreamObserver (because it is still a StreamObserver). We can rename it this is still not good.